system/core/dynamic/BeanPopulator.cfc

```diff
@@ -362,11 +362,15 @@ component {
 						}
 
 						var getEntityMap = function(){
-							if( listFind( "2018,2021", server.coldfusion.productVersion.listFirst() ) ){
+							try {
```

Comment:
I wouldn't try the old approach first. Do the new approach first, because the old one, will only target lucee 5 with the old extension.

Reply:
Updated. Resolved.